README.md

```diff
@@ -764,6 +764,16 @@ NOTE: To upload to the root of a drive, specify its root folder ID as the parent
 ardrive drive-info -d "c7f87712-b54e-4491-bc96-1c5fa7b1da50" | jq -r '.rootFolderId'
 ```
 
+### The IPFS+Arweave integration
+
+If you'd like to share the file you are attempting to upload to the IPFS protocol as well, you can set the `--ipfs` flag to the `upload-file` command.
```

Comment:
That's not true as we're simply adding tags that can be used when querying gql

Reply:
> simply adding tags that can be used when querying gql

And for the IPFS+Arweave nodes to work as a pinning service. So, do we want to explain the part of the GQL tag to the user or by having the link there is just enough?